doc/source/tools/postprocessing/postprocessing.rst

```diff
@@ -0,0 +1,71 @@
+====================================
+Post-processing Lethe with PyVista
+====================================
+
+Lethe has a `post-processing module <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ written in Python based on Lethe users specific needs. `lethe_pyvista_tools <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ was conceived to optimize the reading and post-treatment of Lethe data using Python.
+
+The module is powered by `PyVista <https://docs.pyvista.org/>`_, a powerful 3D plotting and mesh analysis tool. It is a "pythonic" interface to deal wih Visualization Toolkit (VTK) data. The module also uses the powerful `multiprocessing <https://docs.python.org/3/library/multiprocessing.html>`_ Python library to increase post-processing speed by running tasks in multiple processors.

```

Comment:
```suggestion
The module is powered by `PyVista <https://docs.pyvista.org/>`_, a powerful 3D plotting and mesh analysis tool. It is a "pythonic" interface to deal wih Visualization Toolkit (VTK) data. The module also uses the powerful `multiprocessing <https://docs.python.org/3/library/multiprocessing.html>`_ Python library to increase post-processing speed by running tasks in multiple processors.

```
use italic instead of " " for pythonic